rfcs/0006-runtime-references-interface.md

```diff
@@ -44,6 +45,11 @@ its builtin hash scanning and just use the file. Both files are
 newline-separated lists of store paths. Parse errors in the
 `runtime-references` file, or non-existent store paths, fail the build.
 
+To avoid duplicating the default reference scanning behavior in Nix,
+we could also make a tiny tool part of Nix itself that wraps the
+`scanForReferences` API, and then expose that in the `<nix>` namespace
+so tooling can get the full path to that tool.
```

Comment:
Yeah, sounds good. Maybe we can even do `callPackage <nix/store-scanner.nix> {}` to avoid the closure questions? But either way yeah, put something in about that.